_data/providers.json

```
@@ -438,7 +443,7 @@
       "name": "WitherHosting",
       "url": "https://witherhosting.com/",
       "description": "Get Geyser as a plugin. Put the port provided by the host in your config.yml for the Java and bedrock sections, or create a port in the ports manager on the panel and use that for bedrock with the main one for Java, and you will be able to connect! Refer to the hosts [support article](https://support.witherhosting.com/en/article/how-to-install-and-use-geysermc-1xn5l6v/) for more"
-    }
+    },
```

Please remove this unneeded comma at the end of the list
```suggestion
    }
```